README.md

```diff
@@ -1,4 +1,262 @@
 IPFS RFCs
 =====
 
-This repository will be the place where people propose, comment on, and refer to RFCs for changes to IPFS and its surrounding projects. Its initial manifestation will probably look a lot like the [Rust lang RFCs repository](https://github.com/rust-lang/rfcs)
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ipfs-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+IPFS, libp2p or the RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Any semantic or syntactic change to the protocols that is not a bugfix.
+  - Removing protocol features, including those that are feature-gated.
```

Comment:
"feature-gated" in this context means "feature flags" I'm guessing? So even removing experimental features, would need an RFC?

Reply:
are you suggesting a change here?

Reply:
I'm unsure if I want to suggest a change as I'm unsure if "feature-gated" refers to "feature flags" and experimental features. If it does, I'm unsure if we want to go through a RFC process to change/remove them, as they are marked as experimental just because we want the process to be lighter around changing them.

If it's referring to something else, I'd like it to be clarified as I don't know what it means.